docs/install.md

```diff
@@ -48,7 +48,24 @@ Then, install Spark with your custom configuration:
 
 Run the following command to see all configuration options:
 
-    $ dcos package describe spark --config
+$ dcos package describe spark --config
+
+## Customize Spark Distribution
+
+DC/OS Spark does not support arbitrary Spark distributions, but
+Mesosphere does provide multiple pre-built distributions, primarily
+used to select Hadoop versions.  To use one of these distributions,
+first select your desired Spark distribution from here: "<link>", then
```

Comment:
cc @sascala How do I do an internal link?  I want to link to the new spark-versions.md file in this PR.